### PR TITLE
Work on build system

### PR DESCRIPTION
**Pull request by bnavigator**
Reopen of #90 rebased onto master.

This still needs some tweaking with the .travis.yml file.

### CODE REVIEW

File: conda-recipe-openblas/meta.yaml [hunk 1]
```diff
@@ -10,48 +10,30 @@ build:
...
-   # https://go.microsoft.com/fwlink/?LinkId=532606&clcid=0x409
-   
+    - libopenblas
+    - python
+    - scikit-build
+    - pip
+    
   run:
     - python {{ PY_VER }}
     - {{ pin_compatible('numpy') }}
```
Comment (repagh) [inline conda-recipe-openblas/meta.yaml]: need to add - libopenblas here

Comment (bnavigator) [reply conda-recipe-openblas/meta.yaml]: You should be able to write to this branch yourself?

Comment (repagh) [reply conda-recipe-openblas/meta.yaml]: I think I need write access to python-control/Slycot for that...

Would the single failure on Linux with Python 3.5 be fixed with a 
export LD_LIBRARY_PATH="$HOME/miniconda/envs/test-environment/lib:";

OSX  needs 10.9 SDK?

# on OSX, the SDK for 10.9 is currently needed
# download the 10.9 sdk from https://github.com/phracker/MacOSX-SDKs/releases
# unpack and set environment variable CONDA_BUILD_SYSROOT to that location

Comment (murrayrm) [reply conda-recipe-openblas/meta.yaml]: > I think I need write access to python-control/Slycot for that...

Done.

Comment (bnavigator) [reply conda-recipe-openblas/meta.yaml]: the single failure was fixed by adding the BLA_VENDOR variable.

I don't understand the MacOS X setup. Travis has settings for different xcode installations. See also https://github.com/bnavigator/Slycot/tree/work-on-build-plus-travis

Comment (bnavigator) [reply conda-recipe-openblas/meta.yaml]: Continue discussion in #92 and #93

**Pull request merged**